src/main/java/de/chojo/repbot/dao/access/guild/settings/sub/thanking/Channels.java

```diff
@@ -87,14 +83,16 @@ public boolean isEnabledByCategory(@Nullable Category category) {
         return !categories.contains(category.getIdLong());
     }
 
-    public List<TextChannel> channels() {
-        return DiscordResolver.getValidTextChannelsById(guild(), new ArrayList<>(channels));
+    public List<GuildChannel> channels() {
+        return channels.stream().map(guild()::getGuildChannelById)
```

Comment:
That's the only change? The others are mostly formatting.

Reply:
yeah thats the only effective change